source/extensions/filters/common/rbac/engine_impl.cc

```diff
@@ -30,6 +33,11 @@ bool RoleBasedAccessControlEngineImpl::allowed(const Network::Connection& connec
                                                const Envoy::Http::RequestHeaderMap& headers,
                                                const StreamInfo::StreamInfo& info,
                                                std::string* effective_policy_id) const {
+  // Automatically allow if LOG action
```

Comment:
should we instead write to the metadata here instead of inside the filter?

Reply:
Yeah I think that's probably a good solution. I mirrored the other actions which set metadata in the filter. It might be best to set all metadata in the engine, including for ALLOW and DENY since they set the same metadata.